README.md

```diff
@@ -141,6 +141,36 @@ For syntax, see [lookup.json](./lib/lookup.json), the available attributes are:
 If you want to pass options to npm, eg `--registry`, you can usually define an
 environment variable, eg `"npm_config_registry": "https://www.xyz.com"`.
 
+## citgm-stress
+
+//Need to write a description here//
```

Comment:
That is in the TODO list above, I wanted to get everyone's opinion on a suitable description CC @nodejs/citgm